pkg/embed/templates/peer.tmpl

```diff
@@ -1,6 +1,24 @@
 {{ $peer := .Peer }}{{ $peerName := .Name }}{{ $global := .Config }}
 
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# !!!                THIS FILE IS AUTOGENERATED BY PATHVECTOR               !!!
+# !!!  DO NOT EDIT THIS FILE, CHANGES WILL BE OVERRIDEN ON NEXT GENERATION  !!!
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+
+{{ if not (Empty $peer.Comments) }}
```

Comment:
Adding comments to an autogenerated file seems to have limited utility to me. Can you describe your use case for this that wouldn't already be covered by YAML comments in the Pathvector config?

Reply:
This is used internal by other scripts, but since this is optional I don't see the harm in adding this. Additional information in config files could be useful for debuging.

One example of a script is checking if the session is established or not, if it isn't then the scripts looks inside the bird config file for extra comment data like contact information or status url to check is there is maintenance and sends a report that the session is down. Grabbing this data from a extra yml file and filter this data is extra work, it also makes it easy to find the information quickly in the bird config file.